cengage/react-magma
# fix(DatePicker): Date no longer disappears w/ invalid date

Issue: #1269 

## What I did

- When the input loses focus, we no longer clear the entered date. 
- Added an example in Storybook and Doc site of error handling for this scenario

## Screenshots:

![chrome-capture-2024-3-24](https://github.com/cengage/react-magma/assets/91160746/57ad5ea4-0756-48ca-a51a-0a5f034af850)

## Checklist 

## How to test

1. Go to https://next--upbeat-sinoussi-f675aa.netlify.app/api/date-pickers/#minimum_and_maximum_dates
2. Click on the picker and pick any date
3. Now try entering an invalid date (01/07/2020) manually
4. Notice the date does not disappear, and an error message is visible
